astrocut/cube_cut.py

```diff
@@ -450,27 +461,102 @@ def _update_primary_header(self, primary_header):
+        # TODO : The name of FIRST_FFI (and LAST_FFI) is too long to be a header kwd value.
+        # Find a way to include these in the headers without breaking astropy (maybe abbreviate?).
+        # primary_header['FIRST_FFI'] = (self.first_ffi, 'the FFI used for the primary header 
+        # keyword values, except TSTOP')
+        # primary_header['LAST_FFI'] = (self.last_ffi, 'the FFI used for the TSTOP keyword value')
 
         primary_header['RA_OBJ'] = (self.center_coord.ra.deg, '[deg] right ascension')
         primary_header['DEC_OBJ'] = (self.center_coord.dec.deg, '[deg] declination')
 
-        primary_header['TIMEREF'] = ('SOLARSYSTEM', 'barycentric correction applied to times')        
-        primary_header['TASSIGN'] = ('SPACECRAFT', 'where time is assigned')                         
+        timeref = 'SOLARSYSTEM' if product == 'SPOC' else 'None'
+        tassign = 'SPACECRAFT' if product == 'SPOC' else 'None'
+        primary_header['TIMEREF'] = (timeref, 'barycentric correction applied to times')        
+        primary_header['TASSIGN'] = (tassign, 'where time is assigned')
         primary_header['TIMESYS'] = ('TDB', 'time system is Barycentric Dynamical Time (TDB)')
+
         primary_header['BJDREFI'] = (2457000, 'integer part of BTJD reference date')           
         primary_header['BJDREFF'] = (0.00000000, 'fraction of the day in BTJD reference date')    
         primary_header['TIMEUNIT'] = ('d', 'time unit for TIME, TSTART and TSTOP')
 
+        delete_kwds_wildcards = ['SC_*', 'RMS*', 'A_*', 'AP_*', 'B_*', 'BP*', 'GAIN*', 'TESS_*', 'CD*',
+                                 'CT*', 'CRPIX*', 'CRVAL*', 'MJD*']
+        delete_kwds = ['COMMENT', 'FILTER', 'TIME', 'EXPTIME', 'ACS_MODE', 'DEC_TARG', 'FLXWIN', 'RA_TARG',
+                       'CCDNUM', 'CAMNUM', 'WCSGDF', 'UNITS', 'CADENCE', 'SCIPIXS', 'INT_TIME', 'PIX_CAT',
+                       'REQUANT', 'DIFF_HUF', 'PRIM_HUF', 'QUAL_BIT', 'SPM', 'STARTTJD', 'ENDTJD', 'CRM',
+                       'TJD_ZERO', 'CRM_N', 'ORBIT_ID', 'MIDTJD']
+
+        if product == 'TICA':
+
+            # Adding some missing kwds not in TICA (but in Ames-produced SPOC ffis)
+            primary_header['EXTVER'] = ('1', 'extension version number (not format version)')
+            primary_header['SIMDATA'] = (False, 'file is based on simulated data')
+            primary_header['NEXTEND'] = ('2', 'number of standard extensions')
+            primary_header['TSTART'] = (primary_header['STARTTJD'], 'observation start time in TJD of first FFI')
+            primary_header['TSTOP'] = (primary_header['ENDTJD'], 'observation stop time in TJD of last FFI')
+            primary_header['CAMERA'] = (primary_header['CAMNUM'], 'Camera number')
+            primary_header['CCD'] = (primary_header['CCDNUM'], 'CCD chip number')
+            primary_header['ASTATE'] = ('None', 'archive state F indicates single orbit processi')
```

Comment:
Elsewhere (see https://github.com/spacetelescope/astrocut/blob/1431a64a2ef7eebe377bd8fcfecebd2ee0473f79/astrocut/cube_cut.py#L573) these headers are assigned the actual value `None` instead of the string 'None'. Does it matter which is used?

Reply:
In cases where a user wants to index one of these keywords, it might be better for them to see a 'None' string as opposed to nothing (the latter would probably make it seem like the HDU is incomplete, so I'll go ahead and change everything to a string.

Reply:
No, fixed